src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/StartTimeSelection.tsx

```diff
@@ -97,4 +103,7 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
   radioSegments: {
     marginTop: theme.spacings.medium,
   },
+  warningBox: {
```

Comment:
Is this being used?

Reply:
Nope, there was a message for testing, but removed it. Good catch 😄